src/amo/components/SearchSort/SearchSort.scss

```diff
@@ -13,7 +13,7 @@
   background: $base-color;
   border-radius: $border-radius-default;
   border: 0;
-  margin: 0 ($padding-page * 2) $padding-page;
+  margin: ($padding-page * 2) $padding-page;
```

Comment:
This should be `margin: $padding-page ($padding-page * 2);`. The top and bottom should have the regular margins and the left/ride need double the usual margin as it's supposed to be less wide than the cards below it. Removing the `0` means what was previously left/right margin is now top/bottom so the margins will be off 😄 

If you change this to `margin: $padding-page ($padding-page * 2);` that should be good.